Sorting/sorting.hpp

```diff
@@ -20,7 +20,7 @@ class Sort {
     void BubbleSort();
     void InsertionSort();
     void MergeSort();
-    void QuickSort();
+    void QuickSort(int p=0, int r=-10);
```

Comment:
```int r=-10```
Why is hardcoding required? It's a bad practice. Is there some other way we can handle this?

Reply:
@shashwatj07 Okay done.